web/sites/default/settings.php

```diff
@@ -45,3 +45,6 @@
 }
 
 $settings['hash_salt'] = 'CUX2SN2xDPrzIOzft75UAATQY7vrzvgBHzEGWrquM7ZU_4xzOdRj0vRFIYU5tRxYiEqDkxgeKw';
+
+// define $pptstring in config
+$config['texas_asa24.settings']['pptstring'] = 'ID=1256&DEP-F=12W&DEP-L=N&SUSD-F=N&SUSD-L=N&CWWD=B&MAP=N&HTDS-F=N&HTDS-L=N&WAT=N';
```

Comment:
The sites/default directory is the one for SHC, so the pptstring here will be different (including only the CC program initially), and we should be getting that from Meredith.
The value here should be added to the sites/texas/settings.php file.